core/src/main/java/io/questdb/cutlass/line/tcp/LineTcpMeasurementEvent.java

```diff
@@ -48,9 +49,12 @@ class LineTcpMeasurementEvent implements Closeable {
     private final boolean symbolAsFieldSupported;
     private final int maxColumnNameLength;
     private final boolean autoCreateNewColumns;
+    private final int defaultSymbolCapacity;
+    private final boolean defaultSymbolCacheFlag;
     private int writerWorkerId;
     private TableUpdateDetails tableUpdateDetails;
     private boolean commitOnWriterClose;
+    private final AlterOperationBuilder addColumnBuilder = new AlterOperationBuilder();
```

Comment:
should be `alterOperationBuilder`

Reply:
It's only used to add columns, so both names apply. I don't mind renaming it though.

Reply:
Renamed in 0da96063674df9fb51bacc334a93b9d4971f1a16